File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}
        result = []
        for num in nums:
            freq[num] = freq.get(num,0)+1
        freq = dict(sorted(freq.items(), key=lambda x: x[1], reverse=True))
        print(freq)
        result = list(freq.keys())
        return result[:k]

        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        if(nums.length <= 1) return nums.length;
        Set<Integer> s = new TreeSet<>();

        for(int num : nums){
            s.add(num);
        }

        int result =1;
        int temp =1;
        for(int n : s){
            if(s.contains(n+1)){
                temp+=1;
                result = Math.max(temp , result);
            }else{
                temp =1;
            }
        }
        return result;
    }
}
